2-mandatory/1-writers.js

```diff
@@ -70,49 +74,90 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+const logDeadWritersInTheirForties = () => {
+  writers.filter((writer) => {
+    if (writer.age >= 40 && writer.age <= 49 && writer.alive === false)
+      console.log(
+        `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
+      );
+  });
```

Comment:
Performance suggestion. SInce the logical comparison works left to right and fails at the first false comparison. You may want to check whether the author is still alive before checking the age.